src/ray/raylet/scheduling/policy/scheduling_policy_test.cc

```diff
@@ -551,6 +551,79 @@ TEST_F(SchedulingPolicyTest, BundleSchedulingMaxFractionOneCpuReservationGuarant
   ASSERT_TRUE(to_schedule.status.IsSuccess());
 }
 
+// TEST_F(SchedulingPolicyTest, BundleSchedulingMaxFractionOneCpuReservationGuaranteeTest)
```

Comment:
are you going to fix those tests?

Reply:
yes. Let me do that tomorrow. forgot about this...